docs/audit.md

```diff
@@ -179,17 +184,83 @@ While similarly named, IBM's reporting feature fulfills a _different_ use case f
+If a report is run without any `fail-on` arguments (`detect-secrets audit --report .secrets.baseline`), it will execute all the fail checks by default, yet always emit a `0` exit code—even if checks fail.
 
-In CI / CD, it is recommended to provide all `fail-on` args:
+In CI / CD, it is recommended to provide all `fail-on` arguments:
 
 ```shell
-detect-secrets audit --report --fail-on-on-unaudited fail-on-live --fail-on-on-audited-real .secrets.baseline
+detect-secrets audit --report --fail-on-on-unaudited --fail-on-live --fail-on-on-audited-real .secrets.baseline
 ```
 
-TODO: add instructions for setting up in CI / CD
+Below are three documented methods for adding detect-secrets reporting to your pipeline. It is recommended to [use the `detect-secrets` Docker image](#using-detect-secrets-docker-image).
+
+#### Using `detect-secrets` Docker Image
+
+The general-purpose Docker image comes pre-packaged with Python, allowing you to skip the Python installation process.
+
+To use this image in your pipeline, add the following commands to your pipeline script:
+
+1. Get the latest image:
+    - `docker pull ibmcom/detect-secrets:latest`
+2. Mount the directory containing your code to the Docker image's `/code` folder, since it's the working directory for detect-secrets. Pass in the `scan` command to update the baseline file. This file should be up-to-date before a report is run:
+    - `docker run -it --rm -v $(pwd):/code ibmcom/detect-secrets:latest scan --update .secrets.baseline`
+3. Run a report against the updated baseline file:
+    - `docker run -it --rm -v $(pwd):/code ibmcom/detect-secrets:latest audit --report --fail-on-unaudited --fail-on-live --fail-on-audited-real .secrets.baseline`
+
+#### Installing via pip
+
+##### Travis
+
+Add this code to your `travis.yml` file:
+
+```yaml
+language: generic
+sudo: required
+addons:
+    apt:
+        packages:
+            - python3
+            - python3-pip
+            - python3-setuptools
+install:
+    # Required to install detect-secrets
+    - sudo chmod o+rwx /usr/lib/python3/dist-packages/
+    - python3 -m pip install -U pip
+    - python3 -m pip install --upgrade "git+https://github.com/ibm/detect-secrets.git@master#egg=detect-secrets"
+script:
+    # Update the baseline file
+    - detect-secrets scan --update .secrets.baseline
+    # Report with all fail checks
+    - detect-secrets audit --report --fail-on-unaudited --fail-on-live --fail-on-audited-real .secrets.baseline
+```
+
+##### Other pipelines
+
+For other pipelines, repurpose the commands from the [Travis example](#travis). The steps are:
+
+1. Install Python 3 and Pip 3
+2. Install detect-secrets
+3. Scan and update the baseline
+4. Run a report against the baseline
+
+#### Using `detect-secrets-redhat-ubi` Docker Image
+
+This Docker image offers the same benefits as the general-purpose [`detect-secrets` Docker image](#using-detect-secrets-docker-image), but you cannot directly pass in detect-secrets commands.
```

Comment:
Do you thing if would be useful to add a sentence or two explaining why using the redhat image is useful?

Reply:
Definitely, I'll add this in

Reply:
hmm, I think we should discuss this as a team. We are going to be supporting both going forward so it might be useful for us to make that recommendation

Reply:
I think for now, we can merge these changes and then we can decide afterwards